go-licenses.yaml

```diff
@@ -1,18 +1,35 @@
 package:
   name: go-licenses
   version: 1.6.0
-  epoch: 0
+  epoch: 1
   description: A lightweight tool to report on the licenses used by a Go package and its dependencies. Highlight! Versioned external URL to licenses can be found at the same time.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - go
 
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
```

Comment:
```suggestion
```

`go/build` says it needs `go`, and `git-checkout` says it needs `git`, which requires the others.